cmd/traffic/cmd/manager/service.go

```diff
@@ -545,3 +546,27 @@ func (m *Manager) WatchLookupHost(session *rpc.SessionInfo, stream rpc.Manager_W
 func (m *Manager) expire() {
 	m.state.ExpireSessions(m.clock.Now().Add(-15 * time.Second))
 }
+
+// reapDomain informs SystemA that an intercept with a domain has been garbage collected
+func (m *Manager) reapDomain(ctx context.Context, sa systema.SystemACRUDClient, interceptUpdate watchable.InterceptMapUpdate) error {
```

Comment:
Why did you define these here in `service.go` instead of `manager.go` (where it seems to me like they belong?--everything in `service.go` deals directly with responding to gRPC requests).

Reply:
This was bad pattern matching since I saw all the methods in that file and didn't see any in `manager.go` but I will move it.